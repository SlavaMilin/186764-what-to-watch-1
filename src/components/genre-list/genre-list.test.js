import React from 'react';
import renderer from 'react-test-renderer';
import GenreList from "./genre-list";

it(`render genre list`, () => {
  const tree = renderer.create(<GenreList genres={[`first`, `second`]}/>);
  expect(tree).toMatchSnapshot();
});
