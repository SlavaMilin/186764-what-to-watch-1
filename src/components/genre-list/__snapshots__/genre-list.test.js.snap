// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render genre list 1`] = `
<ul
  className="catalog__genres-list"
>
  <li
    className="catalog__genres-item"
  >
    <a
      className="catalog__genres-link"
      href="#"
    >
      first
    </a>
  </li>
  <li
    className="catalog__genres-item"
  >
    <a
      className="catalog__genres-link"
      href="#"
    >
      second
    </a>
  </li>
</ul>
`;
